pkg/asset/templates/content/bootkube/openshift-service-ca-signer-secret.go

```diff
@@ -0,0 +1,64 @@
+package bootkube
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/templates/content"
+)
+
+const (
+	openshiftServiceCASignerSecretFileName = "openshift-service-ca-signer-secret.yaml.template"
```

Comment:
nit: do we need the `openshift` here either?

Reply:
convention per the other yaml templates